draft-ietf-rtcweb-jsep.xml

```diff
@@ -2467,6 +2492,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>If present, a single "a=rtcp-mux" line MUST be parsed as specified in
                 <xref target="RFC5761"/>, Section 5.1.1, and its presence or
                 absence flagged and stored.</t>
+              <t>If present, the "a=rid" lines MUST be parsed as specified in
+	      <xref target="I-D.ietf-mmusic-rid"></xref>, Section 10, and their values stored.</t>
+              <t>If present, the "a=simulcast" lines MUST be parsed as specified
+              in <xref target="I-D.ietf-mmusic-sdp-simulcast"></xref>, and their
+              values stored.</t>
```

Comment:
A simulcast line has both a "send" and "recv" portion.  I think you mean to ignore the send portion from a remote description.  I added a comment to that effect.